tests/src/test/java/alluxio/underfs/UnderStorageSystemInterfaceIntegrationTest.java

```diff
@@ -126,7 +126,19 @@ public void deleteLargeDirectory() throws IOException {
 
     String[] children = config.getChildren();
     for (String child : children) {
-      Assert.assertFalse(mUfs.exists(child));
+      // Retry for some time to allow list operation eventual consistency for S3 and GCS.
+      // See http://docs.aws.amazon.com/AmazonS3/latest/dev/Introduction.html and
+      // https://cloud.google.com/storage/docs/consistency for more details.
+      // Note: not using CommonUtils.waitFor here because we intend to sleep with a longer interval.
+      boolean childDeleted = false;
```

Comment:
if we need a longer interval we could give the sleep interval as an optional argument to `CommonUtils.waitFor`